app.py

```diff
@@ -33,4 +34,19 @@ def index():
     user = User(email)
     db.session.add(user)
     db.session.commit()
+    requests.post(
+        'https://'
```

Comment:
I'd prefer if you use `.format` here (example `https://{}.api.mailchimp.com/3.0/lists/{}/members.format(param1, param2`)